### ISSUE TITLE
Error on applying filters on SQLAlchemy 1.4.0b1

### ISSUE THREAD

**Issue by vikramarsid** [juliotrigo/sqlalchemy-filters#61]
I have encounterred following errors using 
SQLAlchemy 1.4.0b1

```python
query = apply_filters(query, sqla_filters)
File "/opt/venv/lib/python3.8/site-packages/sqlalchemy_filters/filters.py", line 235, in apply_filters
default_model = get_default_model(query)
File "/opt/venv/lib/python3.8/site-packages/sqlalchemy_filters/models.py", line 141, in get_default_model
query_models = get_query_models(query).values()
File "/opt/venv/lib/python3.8/site-packages/sqlalchemy_filters/models.py", line 64, in get_query_models
models.extend(mapper.class_ for mapper in query._join_entities)
AttributeError: 'Query' object has no attribute '_join_entities'
```

**Comment by iyedg** [comment]
I suppose this is due the unification of the Query object API with the Select object in 1.4. For now you could just use sqlalchemy 1.3.23

**Comment by eliranshemtov** [comment]
Now that SQLAlchemy 1.4.20 is officially out... Will this repo get a fix? :/

**Comment by bodik** [comment]
try https://github.com/juliotrigo/sqlalchemy-filters/pull/69

**Comment by eliranshemtov** [comment]
Is it expected to be merged soon? @bodik

**Comment by voney** [comment]
> try #69

I can confirm this works as expected.

**Comment by mvilanova** [comment]
@bodik is there an ETA on when #69 will be merged? Thanks

**Comment by bodik** [comment]
@mvilanova cannot tell, the package is maintained by @juliotrigo

**Comment by munkien** [comment]
@juliotrigo any news on this? 🥇

**Comment by faravinicius** [comment]
alguma novidade ?

**Comment by bodik** [comment]
I'd say that this issue should be resolved with latest release v0.13.0
